src/app/src/components/FacilityListItemsTable.jsx

```diff
@@ -105,6 +105,24 @@ function FacilityListItemsTable({
                 );
             }
 
+            if (item.status === facilityListItemStatusChoicesEnum.MATCHED
+                && item.id === item.matched_facility.created_from_id) {
+                return (
+                    <FacilityListItemsTableRow
+                        key={item.row_index}
+                        rowIndex={item.row_index}
+                        countryName={item.country_name}
+                        name={item.name}
+                        address={item.address}
+                        status={item.status}
+                        handleSelectRow={handleSelectRow}
+                        hover
```

Comment:
Might be worth dropping `hover` here since the other places that's set are for rows which are interactive, and can be expanded on click.